evm/src/main/java/com/horizen/evm/interop/EvmLog.java

```diff
@@ -0,0 +1,19 @@
+package com.horizen.evm.interop;
+
+import com.horizen.evm.utils.Address;
+import com.horizen.evm.utils.Hash;
+import com.horizen.evm.utils.HexBytes;
+
+import java.math.BigInteger;
+
+public class EvmLog {
```

Comment:
Currently, this contains everything the log in Ethereum has to offer. Depending on how we create and store reciepts in the core we might be able to reduce this.

Reply:
Removed all derived fields.